ringing_tutorial/lib/app_keys.dart

```diff
@@ -4,14 +4,17 @@ class AppKeys {
       '{REPLACE_WITH_YOUR_APN_PROVIDER_NAME}';
   static const String androidPushProviderName =
       '{REPLACE_WITH_YOUR_FIREBASE_PROVIDER_NAME}';
+  // Your Stream API key from the Stream Dashboard
 
   static const String user1Id = '{REPLACE_WITH_YOUR_USER_1_ID}';
   static const String user1Name = '{REPLACE_WITH_YOUR_USER_1_NAME}';
   static const String user1Token = '{REPLACE_WITH_YOUR_USER_1_TOKEN}';
+  // Push notification provider names (configure these in Stream Dashboard)
 
   static const String user2Id = '{REPLACE_WITH_YOUR_USER_2_ID}';
   static const String user2Name = '{REPLACE_WITH_YOUR_USER_2_NAME}';
   static const String user2Token = '{REPLACE_WITH_YOUR_USER_2_TOKEN}';
+  // Configure at least two test users for the demo
```

Comment:
These comments are placed wrong. I think 7/12/17 have to go to 2/3/7

Reply:
fixed, I was trying to stage only those lines and keep my local changes but failed :D